build.sbt

```
@@ -171,7 +171,8 @@ def lintingOptions(scalaVersion: String) = {
     "-Wconf:src=*.JavaMtags.scala&msg=parameter value (ctor|method) in method (visitConstructor|visitMethod):silent",
     "-Wconf:src=*.MtagsIndexer.scala&msg=parameter value owner in method visitOccurrence:silent",
     // silence "The outer reference in this type test cannot be checked at run time."
-    "-Wconf:src=.*(CompletionProvider|ArgCompletions|Completions|Keywords|IndentOnPaste).scala&msg=The outer reference:silent"
+    "-Wconf:src=.*(CompletionProvider|ArgCompletions|Completions|Keywords|IndentOnPaste).scala&msg=The outer reference:silent",
+    "-Wconf:src=*.BasePCSuite.scala:silent"
```

```suggestion
    "-Wconf:src=*.BasePCSuite.scala&msg=parameter value (scalaVersion|classpath) in method (extraDependencies|scalacOptions):silent"
```
It will only silence necessary warnings

---

thx! I was a bit lazy to specify it certainly :smile_cat: